pages/login.vue

```diff
@@ -1,52 +1,51 @@
 <template>
-	<section class="w-screen h-screen flex">
-		<div class="w-2/3 flex justify-center items-center bg-amber-400 ">
-			<!-- <img src="~/assets/logos/login.jpg" alt="" width="600px" /> -->
-		</div>
-		<div class="w-1/3 flex justify-center items-start flex-col">
-			<h2 class="text-center text-4xl text-black font-display font-semibold lg:text-left xl:text-5xl
-      xl:text-bold w-2/3 mx-auto">Iniciar sesión</h2>
+	<section class="flex w-screen h-screen">
+		<div class="flex flex-col items-center justify-center w-1/3">
+			<div class="flex flex-col items-center justify-center w-8/12 gap-6">
 
-			<Notification :message="error" v-if="error" />
+				<h2 class="w-full text-3xl font-bold">
+					¡Bienvenido/a!
```

Comment:
Are we adding i18n? If we are adding, it's the perfect moment to add some properties here in a lang library or something. Before it is getting bigger and bigger.

Reply:
No, im goin to research about that and i will include it